specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/ExpressRouteCircuitPeeringGet.json

```diff
@@ -1,55 +1,58 @@
 {
-   "parameters": {
-      "circuitName": "circuitName",
-      "resourceGroupName": "rg1",
-      "api-version": "2018-08-01",
-      "subscriptionId": "subid",
-      "peeringName": "MicrosoftPeering"
-   },
-   "responses": {
-      "200": {
-         "name": "MicrosoftPeering",
-         "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/expressRouteCircuits/circuitName/peerings/MicrosoftPeering",
-         "etag": "W/\"b2a25b98-2e6d-4d46-87f2-089de5f6fdf9\"",
-         "properties": {
-            "provisioningState": "Succeeded",
-            "peeringType": "MicrosoftPeering",
-            "azureASN": 12076,
-            "peerASN": 100,
-            "primaryPeerAddressPrefix": "123.0.0.0/30",
-            "secondaryPeerAddressPrefix": "123.0.0.4/30",
-            "primaryAzurePort": "A51-TEST-06GMR-CIS-1-PRI-A",
-            "secondaryAzurePort": "A51-TEST-06GMR-CIS-2-SEC-A",
-            "state": "Enabled",
-            "vlanId": 300,
-            "gatewayManagerEtag": "103",
-            "lastModifiedBy": "Customer",
-            "microsoftPeeringConfig": {
-               "advertisedPublicPrefixes": [
-                  "123.1.0.0/24"
-               ],
-               "advertisedCommunities": [],
-               "advertisedPublicPrefixesState": "ValidationNeeded",
-               "customerASN": 23,
-               "legacyMode": 0,
-               "routingRegistryName": "ARIN"
-            },
-            "ipv6PeeringConfig": {
-               "primaryPeerAddressPrefix": "3FFE:FFFF:0:CD30::/126",
-               "secondaryPeerAddressPrefix": "3FFE:FFFF:0:CD30::4/126",
-               "state": "Enabled",
-               "microsoftPeeringConfig": {
-                  "advertisedPublicPrefixes": [
-                     "3FFE:FFFF:0:CD31::/120"
-                  ],
-                  "advertisedCommunities": [],
-                  "advertisedPublicPrefixesState": "ValidationNeeded",
-                  "customerASN": 23,
-                  "legacyMode": 0,
-                  "routingRegistryName": "ARIN"
-               }
+    "parameters": {
+        "circuitName": "circuitName",
+        "resourceGroupName": "rg1",
+        "api-version": "2018-08-01",
+        "subscriptionId": "subid",
+        "peeringName": "MicrosoftPeering"
+    },
+    "responses": {
+        "200": {
+            "name": "MicrosoftPeering",
```

Comment:
Microsoft peering will not have expressroute gw connection. Only private peering. Need to change the example